doc/material/v1.1.0.rst

```diff
@@ -0,0 +1,83 @@
+.. _whatsnew_0100:
+
+Version 1.1.0 (August 27, 2024)
+----------------------------------
+
+{{ header }}
+
+
+This is a release from MESSAGEix-Materials 1.0.0 and includes model additions and enhancements.
```

Comment:
This phrasing sounds odd to me, why would a release of "MESSAGEix-Materials 1.0.0" have a separate version number? I would likely just use "MESSAGEix-Materials".